google-cloud-core/lib/google/cloud/errors.rb

```diff
@@ -48,7 +48,7 @@ def initialize msg = nil
       #
       # @return [Object, nil]
       def status_code
-        return nil unless cause && cause.respond_to?(:status_code)
+        return nil unless cause&.respond_to? :status_code
```

Comment:
I don't mind these, but I thought there were others among us who had a different opinion?

Reply:
There were mixed feelings. There are a few rules that are different for samples and regular code. This was deliberately one of them, but we can change that whenever.

Reply:
FWIW, I strongly prefer `cause&.respond_to? :status_code`.